src/config.js

```diff
@@ -88,6 +88,7 @@ const highlighted_entity_icon = L.icon({
 })
 // How many extra tiles to display around the highlighted selected entries
 const highlighted_entity_margin = 5
+const highlighted_entity_zindex = 100
```

Comment:
How about

const max_entity_width = 10
const zindex_offset_multiplier = max_entity_width*tile_width*(max_zoom - max_native_zoom)**2
const highlighted_entity_zindex = max_entity_width**2*zindex_offset_multiplier

Reply:
Instead of
`max_entity_width*tile_width*(max_zoom - max_native_zoom)^2`
I did
`max_entity_width*tile_width*2^(max_zoom - max_native_zoom)`
Because each zoom level is x2 scale. I think this is right?